app/assets/javascripts/display.js

```diff
@@ -98,99 +109,63 @@ $(document).ready(function(){
-			//this is specific JS for the multi-line view
-			console.log('welcome to multi-line mode');
-			// //template views
+			//MORE TEMPLATING - can be moved to the controller -!!!-
 			var tLine = _.template($('#line-template-display-multi').html());
 
 			$lineCont = $('#line-holder-display-multi');
 
-
-			//seed sequence #0 with a blank line
-			$lineCont.append(
-				tLine({
-					"character":'',
-				    "id": 0,
-				    "sequence": 0,
-				    "content_type": "Non-Dialogue",
-				    "content_text": " ",
-				    "color": null,
-				    "visibility": false,
-				    "created_at": " ",
-				    "updated_at": " "
-					})
-				);
-
 			//build the line dom
 			buildLinesDisp($lineCont);
-			$linesDisp = $('.line-display-multi');
+			//END MORE TEMPLATING - can be moved to the controller -!!!-
 
+			$('.line-display-multi').first().addClass('focus-multi');
 
-			//set first interval
+			//recursive function that scrapes the line sequence number checks for blackouts
 			function heartbeat(){
-				var bottomPad = 65;
+				var bottomPad = 65; //may be able to get rid of -!!!-
 				//ajax goes here next timeout
-				$.ajax('/display/current',
-				  {
-						data: {operator: operator},
-					  dataType: 'json',
-						success:(function(j){
-							console.log('sequence scraped ' + j);
-							if(current!=j){
-								//console.log(j);
-								current=j;
-
-								if(current!==0){
-									$('#shade-multi').fadeOut(dispFadeSpd);
-									var newElem = _.find($linesDisp, function(q){
-											return parseInt($(q).attr('data-sequence'))==j;
-									});
-									$('#body-display-index').stop().animate({scrollTop:$(newElem).position().top-$(window).height()/2+$(newElem).height()+bottomPad}, displayScrollSpd);
-									//remove the focus class
-									$('.focus-multi').removeClass('focus-multi');
-									//console.log('class-removed');
-
-									$(newElem).addClass('focus-multi');
-											//$(this).addClass('shown-display');
-									setTimeout(function(){
-										heartbeat();
-									}, refresh);
-								}else{
-									$('#shade-multi').fadeIn(dispFadeSpd);
-									setTimeout(function(){
-										heartbeat();
-										}, refresh);
-								}
-
-
-									//});
-
-							}else{
-								setTimeout(function(){
-									heartbeat();
-									}, refresh);
-							}
-
-
-						}),
+				$.ajax('/display/current', {
+					data: { operator: operator },
+					dataType: 'json',
+					success:(function(j) {
+						console.log('sequence scraped ' + j);
+						//if the data sequence changed
+						if($('.focus-multi').attr('data-sequence') != j) {
+							console.log("data sequence changed");
+
+							//calculate scroll speed
+							var now = (new Date).getTime();
+							displayScrollSpd = Math.max(MINSCROLLDURATION, (Math.min(MAXSCROLLDURATION, (now - lastScrollMS))));
+							lastScrollMS = now;
+							console.log(displayScrollSpd);
+
+							$('#shade-multi').fadeOut(dispFadeSpd);

```

Comment:
Any jquery selector that pulls from ID here can be optimized by storing it in a variable for faster retrieval. For example:
```
//earlier
var $shadeMulti = $('#shade-multi');
//later
$shadeMulti.fadeOut(dispFadeSpd);
```
Didn't have the opportunity to make these optimizations for display views.